### PR TITLE
Decades marked with red line. Issue #19

### PR DESCRIPTION
**Pull request by thorn573**
Line across decade ring colored red. Disabled annotations b/c change automates their purpose.

### CODE REVIEW

File: leaflet-treering.js [hunk 1]
```diff
@@ -127,13 +127,16 @@ function LTreering (viewer, basePath, options) {
...
+      map.removeLayer(this.annotationAsset.markerLayer);
     } else {
       this.popout.btn.addTo(this.viewer);
       this.viewData.btn.addTo(this.viewer);
       this.ioTools.bar.addTo(this.viewer);
       this.settings.bar.addTo(this.viewer);
-      //defaults overlay 'points' option to disabled
+      //defaults overlay 'points' & 'annotations' option to disabled
       map.removeLayer(this.visualAsset.markerLayer);
+      map.removeLayer(this.annotationAsset.markerLayer);
```
Comment (cmcfadden) [inline leaflet-treering.js:139]: if you're removing the annotation layer in both cases, move it outside the conditional

File: leaflet-treering.js [hunk 2]
```diff
@@ -835,6 +839,12 @@ function VisualAsset (Lt) {
       } else {
         var color = '#00838f';
       }
+
+      //mark decades with red line
+      if ((pts[i].earlywood && pts[i].year % 10 == 0) || (!pts[i].earlywood && pts[i].year % 10 == 0)) {
```
Comment (cmcfadden) [inline leaflet-treering.js:844]: I think this conditional just reduces down to if(pts[I].year %10 ==) right? Do you need to check earlywood and !earlywood?

Comment (thorn573) [reply leaflet-treering.js:843]: I don't. I originally wrote it for just latewood and reused code without thinking too much about it.

**Comment by cmcfadden** [comment]
looks good!

**Pull request merged**